tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -958,39 +959,63 @@ static inline struct mount *real_mount(struct vfsmount *mnt)
     return container_of(mnt, struct mount, mnt);
 }
 
-static __always_inline u32 get_inet_rcv_saddr(struct inet_sock *inet)
+static __always_inline u16 get_inet_sport(struct inet_sock *inet)
 {
-    return READ_KERN(inet->inet_rcv_saddr);
+    return READ_KERN(inet->inet_sport);
 }
 
-static __always_inline u32 get_inet_saddr(struct inet_sock *inet)
+static __always_inline struct sock* get_socket_sock(struct socket *socket)
 {
-    return READ_KERN(inet->inet_saddr);
+    return READ_KERN(socket->sk);
 }
 
-static __always_inline u32 get_inet_daddr(struct inet_sock *inet)
+static __always_inline struct sock_common * get_sock_sk_common(struct sock *sock)
 {
-    return READ_KERN(inet->inet_daddr);
+    // get sock_common from map to avoid reaching bpf stack limit
+    u32 idx = 0;
+    struct sock_common *sk_common = bpf_map_lookup_elem(&sock_common_struct, &idx);
+    if (sk_common == NULL)
+        return NULL;
+
+    #ifdef CORE
+        bpf_core_read((void *)sk_common, sizeof(struct sock_common), &sock->__sk_common);
```

Comment:
Note that now you have the WRONG size of struct sock_common (because you removed the extra fields)